your_daily_dose_malware/main.py

```diff
@@ -1,31 +1,42 @@
 import click
 from .commands.scraper import scraper
-from .commands.utils.retrieve_malware import all_most_recent, hundred_most_recent
+from .commands.utils.retrieve_malware import (all_most_recent,
+                                              hundred_most_recent)
 from dotenv import load_dotenv
 import os
+
 load_dotenv()
 
+
 @click.command(
     help="""
-    Download eitheir hundred recent malwares uploaded within the last 60 min or  all the recent malwares uploaded within the last 48 hours sha256 hash by using api from 'https://mb-api.abuse.ch/api/v1/' 
+    Download either hundred recent malware's uploaded within the last 60 min or
```

Comment:
This help will likely change once we start adding more backends. 
This is very descriptive of Malware Bazaar but keep in mind this tool will likely use 8 or 9 different backends, so we wont be able to go into that much detail for each backend.